phpstan.neon

```diff
@@ -552,3 +552,8 @@ parameters:
                 - rules/Php74/Rector/LNumber/AddLiteralSeparatorToNumberRector.php
 
         - '#^Cognitive complexity for "Rector\\CodingStyle\\Naming\\NameRenamer\:\:renameNameNode\(\)" is 13, keep it under 9$#'
+
+        -
+            message: '#The string value "" is repeated 6 times\. Refactor to enum to avoid typos and make clear allowed value#'
+            paths:
+                - packages/BetterPhpDocParser/PhpDocManipulator/PhpDocTagRemover.php #45
```

Comment:
Thanks, should be fixed in next simplify release :+1:

Reply:
removed at https://github.com/rectorphp/rector-src/pull/805